app/src/main/java/campus/tech/kakao/map/MainActivity.kt

```diff
@@ -47,27 +78,68 @@ class MainActivity : AppCompatActivity() {
 
                 override fun onMapError(error: Exception) {
                     // 인증 실패 및 지도 사용 중 에러가 발생할 때 호출
+                    Log.d("KakaoMap", "onMapError")
                     Log.e("KakaoMap", "onMapError: ", error)
+                    val intent = Intent(this@MainActivity, MapErrorActivity::class.java)
+                    intent.putExtra("error", error.toString().substring(20))
+                    startActivity(intent)
+                    finish()
                 }
             },
             object : KakaoMapReadyCallback() {
                 override fun onMapReady(map: KakaoMap) {
                     // 정상적으로 인증이 완료되었을 때 호출
                     // KakaoMap 객체를 얻어 옵니다.
                     kakaoMap = map
+                    //카메라 업데이트
+                    var cameraUpdate =
+                        CameraUpdateFactory.newCenterPosition(LatLng.from(latitude, longitude))
+                    kakaoMap?.moveCamera(cameraUpdate)
+                    addMarker(map, latitude, longitude, name)
+
                 }
-            })  //mapView.start
+            }
+        )  //mapView.start
 
     }   //onCreate
 
     override fun onResume() {
         super.onResume()
         mapView?.resume()
+        Log.d("KakaoMap", "onMapResume")
+
     }
 
     override fun onPause() {
         super.onPause()
         mapView?.pause()
+        Log.d("KakaoMap", "onMapPause")
     }
 
+    private fun addMarker(kakaoMap: KakaoMap, latitude: Double, longitude: Double, name: String) {
+        val labelManager = kakaoMap.labelManager
+
+        if (labelManager == null) {
+            Log.e("AddMarker", "LabelManager is null")
+            return
+        }
```

Comment:
만약 Log를 더이상 확인할 필요가 없어질 경우에는 이렇게 작성해볼 수도 있어요.

```suggestion
        val labelManager = kakaoMap.labelManager ?: return
```